pkgs/by-name/te/tera-cli/package.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "tera-cli";
+  version = "0.2.5";
+
+  src = fetchFromGitHub {
+    owner = "chevdor";
+    repo = pname;
+    rev = version;
+    hash = "sha256-W+pcVLxOlikwAGvx0twm23GyCMzdqnHY0YBNtcsSB5I=";
+  };
+
+  cargoHash = "sha256-A01mok8KQk1FV8P7E4svdBCW6xqpduHy1XuUcdDFjfc=";
+
+  meta = with lib; {
+    description = "A command line utility written in Rust to render templates from json|toml|yaml && ENV, using the tera templating engine";
```

Comment:
```suggestion
    description = "A command line utility to render templates from json|toml|yaml && ENV, using the tera templating engine";
```

Reply:
This is the project's original description from https://github.com/chevdor/tera-cli/blob/9f845e7ed47efcc434e66098f279da08146bf1e2/Cargo.toml#L3 - is there a reason not to use this?

Reply:
`meta.description` is for a brief description of what the package does, and there I don't see what language a package is written in a good fit. (: 

[https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/meta.chapter.md#description-var-meta-description](https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/meta.chapter.md#description-var-meta-description)